lntest/harness_assertion.go

```diff
@@ -251,19 +252,33 @@ func (h *HarnessTest) AssertNumEdges(hn *node.HarnessNode,
 		old = hn.State.Edge.Total
 	}
 
+	// filterDisabled is a helper closure that filters out disabled
+	// channels.
+	filterDisabled := func(edge *lnrpc.ChannelEdge) bool {
```

Comment:
So we cannot garantee that the node will be removed in the graph, but rather need to look at the disable bit. There is no way to remove the edge from the db ?

Reply:
From a previous test, for non-standby nodes, when they forgot to close their channels, the edges will be kept in the standby nodes' graphs, then they will be marked as disabled, which is the expected behavior for `lnd`. However, this setup is really annoying as we now need to carry on state changes made from other tests, but soon this will not be the case as I think all the itest flakes are fixed in the final PR of blockbeat.

Reply:
Thanks for that clarification!